tallyhowallet/extension
# Calculate total NFTs floor price including assets other than ETH

Resolves https://github.com/tallyhowallet/extension/issues/2775

### What
- The total floor price should sum all prices we know about, if they are using different currencies for the floor price let's calculate its USD and ETH values and add them to the total sum.
- Let's sort NFTs by price regardless of which currency is used
- Minor UI improvements: showing price tag on badges list, fix for radio button cursor

### Testing
1. Activate Avalanche and BSC chains for wallets that have some NFTs on these chains
2. Check if total floor price in the header of NFTs page includes prices of collections with floor prices in AVAX or BNB
3. Sort collections by price - should sort comparing USD value of currencies

Latest build: [extension-builds-2885](https://github.com/tallyhowallet/extension/suites/10463880816/artifacts/518165647) (as of Thu, 19 Jan 2023 12:44:15 GMT).
